### DISCUSSION TITLE
Remove server middleware return statement - Sever middleware shouldn't return anything

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxtrdev/nuxtr-vscode#133] [Questions]
![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/10191085/8b839ca7-c051-4579-b9ab-d5fdb8a1590b)
Before fix:
![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/10191085/7d36a618-6099-4abb-9e31-bd6afe6172a3)
After fix:
![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/10191085/2d2021de-d1dc-4bad-8458-3e9726aa23c6)

**Comment by adhamfarrag** [comment]
Hi @angelhdzmultimedia, Thanks for bringing this to my attention. This must have bee nadded to the default snippet after we added it to Nuxtr. Will update it soon :)
